index.ts

```diff
@@ -84,73 +84,58 @@ class ServerlessCustomDomain {
      * Wraps creating a domain and resource record set
      */
     public async createDomain(): Promise<boolean> {
-        try {
-            const certArn = await this.getCertArn();
-            const domainInfo = await this.createCustomDomain(certArn);
-            await this.changeResourceRecordSet("UPSERT", domainInfo);
-            this.serverless.cli.log(`Custom domain ${this.givenDomainName} was created/updated.
-            New domains may take up to 40 minutes to be initialized.`);
-            return true;
-        } catch (error) {
-            return false;
-        }
+        const certArn = await this.getCertArn();
+        const domainInfo = await this.createCustomDomain(certArn);
+        await this.changeResourceRecordSet("UPSERT", domainInfo);
+        this.serverless.cli.log(
+            `Custom domain ${this.givenDomainName} was created/updated.
+            New domains may take up to 40 minutes to be initialized.`,
+        );
+        return true;
```

Comment:
Why return true here?

Reply:
Returned true for unit testing. Now testing by matching console.log calls

Reply:
Now removing all unnecessary returns